src/components/UserList.jsx

```diff
@@ -46,11 +66,18 @@ class UserList extends React.PureComponent {
       users,
     } = this.props;
 
+    const { isAccordionExpanded } = this.state;
+
     return (
-      <div className={classes.root}>
-        <div className={classes.header}>
-          {typeOfUsers}:{' '}
-          {horizontal && canManageUsers && (
+      <Accordion expanded={isAccordionExpanded} onChange={this.toggleAccordion}>
```

Comment:
Is the 'isAccordionExpanded' state used by anything else?  I think you might be able to remove expanded, onChange and the state tracking of this Accordion.

The starting state of the current UI allows users to see the user lists without clicking.  Should these be expanded by default?  defaultExpanded should do the trick, but I suggest checking with Lou for the UX decision.

Reply:
I have asked Lou for what to do with the accordions initial state!

Reply:
Lou responded with:

First accordion starts open, to show the user the behavior possible. The other two remain closed until user interacts.

I've updated the PR to match!

Reply:
Are you trying to cover a case that "defaultExpanded" doesn't cover? [See Props](https://mui.com/material-ui/api/accordion/#:~:text=for%20more%20details.-,defaultExpanded,-bool).  If you use that, you can eliminate the need to track the expanded state yourself and just leverage the property passed in.  eg. defaultExpanded={prop.defaultOpen}.

Reply:
wow ha ha, yup, that totally works in the props - pushed up a  fix to clean this up!